functions/_fzf_preview_changed_file.fish

```diff
@@ -0,0 +1,24 @@
+function _fzf_preview_changed_file
+    set -l path (string split ' ' $argv)[-1]
+    if string match -r '^\?\?' $argv >/dev/null
+        echo Untracked
+        echo
+        _fzf_preview_file $path
```

Comment:
niceeeee, so clever to go call _fzf_preview_file
can you go to the top of _fzf_preview_file and remove the comment?

Reply:
is this? 
`# helper function for _fzf_search_directory`

Reply:
yup, actually can you update it to say `helper function for _fzf_search_directory and _fzf_preview_changed_file`

Reply:
> yup, actually can you update it to say `helper function for _fzf_search_directory and _fzf_preview_changed_file`

emm, you mean `_fzf_search_directory` and `_fzf_search_git_status` right?